src/voxel.h

```diff
@@ -231,60 +229,55 @@ class VoxelArea
 
 		assert(contains(a));	// pre-condition
 
+		const auto &take = [&result] (v3s16 min, v3s16 max) {
+			if (max >= min) {
```

Comment:
FYI: This operator does not compare all coordinates individually.

Reply:
huh? what else would it do?
![grafik](https://github.com/user-attachments/assets/e6be2039-a560-42fe-98d4-9faae2a0155a)

Reply:
It sorts like a tuple, lexicographically. This gives a total order or so, which you need e.g. for std::map.

Reply:
`v3s16(1,2,3) < v3s16(2,1,3)`
Why not sort each component individually?

Reply:
`max.X >= min.X && max.Y >= min.Y && max.Z >= min.Z` is probably what you intended, thus my comment.

But why exactly is `(b.getVolume() != 0)` not sufficient? Due to potential integer under-/overflows of the edge coordinates?

Reply:
It's not about integer overflow, even just the simple test I wrote triggers the assert. 

I'll fix this in a different way.